watertap/unit_models/tests/test_gac.py

```diff
@@ -0,0 +1,143 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# 'https://github.com/watertap-org/watertap/'
+#
+###############################################################################
+
+from pyomo.environ import (
+    ConcreteModel,
+    SolverFactory,
+    TerminationCondition,
+    TransformationFactory,
+)
+from pyomo.network import Arc
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.generic_models.unit_models import Feed, Product
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.scaling import calculate_scaling_factors
+import idaes.core.util.scaling as iscale
+from idaes.core.util.tables import (
+    create_stream_table_dataframe,
+    stream_table_dataframe_to_string,
+)
+
+from watertap.property_models.ion_DSPMDE_prop_pack_GACadjusted import (
+    DSPMDEParameterBlock,
+)
+from watertap.unit_models.gac import GAC
+
+
+def main():
```

Comment:
@hunterbarber Please have a look at other example test files and try following one of those examples to make your test file. This is useful but we'd consider this to be more of simulation file.

Reply:
Okay, I understood that this wasn't the convention but did need a location to house this early testing prior to the more detailed testing.

Reply:
> Okay, I understood that this wasn't the convention but did need a location to house this early testing prior to the more detailed testing.

@hunterbarber as a quick check, if you rename this function from `main()` to something starring with `test_` e.g. `test_gac_main()`, it will be picked up by pytest and counted in the coverage report.